components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/utils/JWTUtil.java

```diff
@@ -74,6 +75,17 @@ public static boolean handleScopeValidation(HashMap<String,Object> message, Sign
                     .map(s -> s.replace(APIConstants.URN_CHOREO + orgId + ":", ""))
                     .toArray(size -> new String[size]);
             oauthTokenInfo.setScopes(scopes);
+            // check whether organization claim value and orgId matches
+            JSONObject orgClaim = signedJWTInfo.getJwtClaimsSet().getJSONObjectClaim("organization");
+            String orgUuid = orgClaim.getAsString("uuid");
+            if (orgId == null) {
+                log.error("Organization is not present in the request");
+                return false;
+            }
```

Comment:
Shall we move this to around line 72 just after `String orgId = (String) message.get(RestApiConstants.ORG_ID);`

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/11670/commits/a4afd58620f09a8ccb5da7c5822e414948bf6e80